benchmarks/Microsoft.AspNetCore.SignalR.Microbenchmarks/CoreConfig.cs

```diff
@@ -1,34 +0,0 @@
-﻿using BenchmarkDotNet.Columns;
-using BenchmarkDotNet.Configs;
-using BenchmarkDotNet.Diagnosers;
-using BenchmarkDotNet.Engines;
-using BenchmarkDotNet.Jobs;
-using BenchmarkDotNet.Validators;
-
-namespace Microsoft.AspNetCore.SignalR.Microbenchmarks
-{
-    public class CoreConfig : ManualConfig
```

Comment:
Is this configuration no longer required, or did the logic move somewhere else?

Reply:
Moved to common: https://github.com/aspnet/Common/pull/303